postgres/datadog_checks/postgres/util.py

```diff
@@ -557,6 +557,23 @@ def trim_leading_set_stmts(sql):
 """.strip(),
 }
 
+QUERY_PG_WAIT_EVENT_METRICS = {
```

Comment:
We can do the same aggregations (and more) with samples in the UI. The issue is that we’re not collecting samples for background processes. I assume the reason for adding these metrics is to get visibility into them. Would it be better to include background processes in sampling instead?

Reply:
Yeah, the system processes are not visible and having visibility on the recovery process would have helped a lot during recent incidents. Also, wait events from sampling is not a metric and the 15 days retention make it hard to use.
If that can be done with samples, that would be great but AFAIK, the last attempt failed years ago? And that wouldn't solve the 15 days retention issues.